.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
+
+enabled:
+  - align_double_arrow
+  - newline_after_open_tag
+  - ordered_use
+  - long_array_syntax
+  - php_unit_construct
+  - php_unit_strict
+
+disabled:
+  - psr0
```

Comment:
A little painful isn't it? Should be deprecated, not broken.

BTW, `psr0` fixer is still supported by php-cs-fixer AFAIK.